openfl/interface/interactive_api/experiment.py

```diff
@@ -357,7 +357,7 @@ def start(
         else:
             self.logger.info("Experiment could not be submitted to the director.")
 
-    def define_task_assigner(self, task_keeper, rounds_to_train):  # noqa: C901
```

Comment:
The Interactive API being slated for deletion, I suggest not to address any code issues there.

Reply:
OK